build/azure-pipelines/linux/sql-product-build-linux.yml

```diff
@@ -147,24 +147,23 @@ steps:
       ./build/azure-pipelines/linux/createDrop.sh
     displayName: Create Drop
 
-  - task: PublishBuildArtifacts@1
-    displayName: 'Publish Artifact: drop'
+  - task: CopyFiles@2
+    displayName: 'Copy Extension Unit Test Coverage Files to: $(Build.ArtifactStagingDirectory)'
+    inputs:
+      SourceFolder: '$(Build.SourcesDirectory)/extensions'
+      Contents: '*/coverage/**'
+      TargetFolder: '$(Build.ArtifactStagingDirectory)/test-results/coverage'
 
   - task: PublishTestResults@2
     displayName: 'Publish Test Results test-results.xml'
     inputs:
-      testResultsFiles: 'test-results.xml'
-      searchFolder: '$(Build.SourcesDirectory)'
+      testResultsFiles: '*.xml'
+      searchFolder: '$(Build.ArtifactStagingDirectory)/test-results'
     continueOnError: true
     condition: and(succeeded(), eq(variables['RUN_TESTS'], 'true'))
 
-  - task: PublishCodeCoverageResults@1
-    displayName: 'Publish code coverage from $(Build.SourcesDirectory)/.build/coverage/cobertura-coverage.xml'
-    inputs:
-      codeCoverageTool: Cobertura
-      summaryFileLocation: '$(Build.SourcesDirectory)/.build/coverage/cobertura-coverage.xml'
-      reportDirectory: '$(Build.SourcesDirectory)/.build/coverage'
-    continueOnError: true
+  - task: PublishBuildArtifacts@1
```

Comment:
why are you changing this? This is how we get the coverage numbers in vso, and you can always see the files as an artifact of the build.

Reply:
we are not running unit tests in this job see the job definition as well as the related warning produced by this step in my screenshots, also, even if we are producing coverage results we can't use this task due to the limitation of ADO pipeline, see the issue description for more details, in a separate PR later i will add a separate step to merge the coverage results produced by all jobs.

Reply:
I still think it makes more sense to have 1 coverage report and if people want to isolate what numbers they see they can filter the report.

Reply:
Specially if we eventually have a way to comment code coverage changes on PR's we will need to have 1 report rather than like 10 different reports.

Reply:
as i mentioned in the issue description, my next tasks in the next couple weeks would be:
1. merge all the results together and use publish coverage result task to publish it. 
1. hook up the build pipeline with coveralls to track the change and the history of code coverage.

Reply:
Doesnt the code coverage task automatically merge them already?

Reply:
no.

https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-code-coverage-results?view=azure-devops#is-code-coverage-data-merged-when-multiple-files-are-provided-as-input-to-the-task-or-multiple-tasks-are-used-in-the-pipeline

![Screen Shot 2020-02-24 at 10 22 30 AM](https://user-images.githubusercontent.com/13777222/75732323-0916bb80-5ca7-11ea-9ba8-460cfa99354b.png)